### DISCUSSION TITLE
First Bullseye-based stable release of free and open source BBN Marine OS for Raspberry Pi is available for downloads.

### DISCUSSION THREAD

**Discussion by mgrouch** [bareboat-necessities/lysmarine_gen#232] [General]
First Bullseye-based stable release of free and open source BBN Marine OS for Raspberry Pi is available for downloads.

This release is arm64 with noticeable improvements in performance.

There is number of new features (compared to previous Buster-based releases)

Better PyPilot support
Better Victron support
Few additional apps

Many included plugins for SignalK and OpenCPN

Fair Winds!

Download link:
https://github.com/bareboat-necessities/lysmarine_gen/releases/tag/v2022-09-09

**Comment by wsturner88** [comment]
I am running an earlier version, and this resolves the issue with the Null data causing OpenCPN to Crash!   Thank you!
Working on the Victron and I found one bug - VE Direct to Signal K - Status:Failed to start: Delimiter is not a constructor

I am Downloading the 9/9 version and will give that a try..

**Reply by mgrouch** [reply]
There is 2022-09-11 version already. 

Did you try Victron web UI which under ‘power’ icon?

**Reply by wsturner88** [reply]
Yes, would not work, I presume because there was no data..  I feed the Rpi via USB and the VE Direct to signal K is what receives the USB feed into SignalK..    Sorry to put this in discussion, should I start a feed under issues?

**Reply by mgrouch** [reply]
> Yes, would not work, I presume because there was no data.. I feed the Rpi via USB and the VE Direct to signal K is what receives the USB feed into SignalK.. Sorry to put this in discussion, should I start a feed under issues?

If you having issue with  VE Direct SignalK plugin you should report it under its project in GitHub
Is it this one you are using?  https://github.com/sbender9/signalk-venus-plugin
There is also SignalK support channel (quite active) on slack http://slack-invite.signalk.org/

**Reply by wsturner88** [reply]
I would, but it is only under your build that I have issues...   it worked in your previous builds when I installed it manually instead of it being included in your build.      I have yet to try your latest release, I plan to do that tonight.   When I am at the boat the wifi is so bad I cant DL the images.

**Reply by mgrouch** [reply]
In SignalK  there is a way to remove a plugin and install it again. Does it make plugin work?

**Reply by mgrouch** [reply]
"Delimiter is not a constructor" issue. Could it be same as in:

https://github.com/W-Geronius/signalk-hwt901b-imu/issues/6

It seems I have fixed my local copy by changing this row in index.js =>
`const DelimiterParser = require('@serialport/parser-delimiter')`
with the following =>
`const { DelimiterParser } = require('@serialport/parser-delimiter')`

**Comment by mpolazzo** [comment]
Hi, I have seen the release announcement. Is there any way to update an existing image of BBNos instead of re-flashing from scratch (without consequent loss of existing parametrization and customization)?
Many thanks.

**Reply by mgrouch** [reply]
Script your customization steps and store them somewhere in GitHub. 
Load new BBN OS image and reapply your customization from your script.

**Reply by mpolazzo** [reply]
Interesting. Thanks. Unfortunately I do have much more experience with Windows programming than Pi. I'll learn...
